jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/GradleLayerConfigurationsTest.java

```diff
@@ -79,6 +81,17 @@ public String getDisplayName() {
     public Set<File> getFiles() {
       return files;
     }
+
+    @Override
+    public TaskDependency getBuildDependencies() {
```

Comment:
I thought this was a little strange, but there isn't really an easy way to Mock this out :disappointed: 
No action required on this comment.